tests/qam-updinstall/update_install.pm

```diff
@@ -87,25 +87,34 @@ sub run {
 
     zypper_call(q{mr -d $(zypper lr | awk -F '|' '/NVIDIA/ {print $2}')}, exitcode => [0, 3]);
 
-    fully_patch_system;
-
     # Get packages affected by the incident.
     my @packages = get_incident_packages($incident_id);
 
     # Extract module name from repo url.
     my @modules = split(/,/, $repos);
-    s{http.*SUSE_Updates_(.*)/}{$1} for @modules;
+    foreach (@modules) {
+        if (s{http.*SUSE_Updates_(.*)/?}{$1}) {
+            next;
+        }
+        else {
+            die "Modules regex failed. Modules could not be extracted from repos variable.";
+        }
+    }
```

Comment:
```suggestion
    foreach (@modules) {
        next if s{http.*SUSE_Updates_(.*)/?}{$1};
        die 'Modules regex failed. Modules could not be extracted from repos variable.';
    }
```

Reply:
Actually it's not minor! The reason the bug exists is because I tried it that way and failed. Thanks for showing me the right way!